LibreHardwareHelper/LibreHardwareHelper/Model/HardwareData/CPU/CpuPower.cs

```diff
@@ -24,6 +24,7 @@ public CpuPower(IHardware cpu, LibreHardwareHelper helper)
         foreach (var s in cpu.Sensors)
             switch (s.Name)
             {
+                case "Package":
```

Comment:
Something I think we should start doing is adding a comment next to each case just saying what it is for. Just so we know in the future

Reply:
agreed, i do wonder if there is a better way to do this, instead of a string on name. something to look for another day